package.json

```diff
@@ -108,5 +108,8 @@
   "volta": {
     "node": "20.10.0",
     "npm": "10.2.4"
+  },
+  "dependencies": {
+    "ebnf": "^1.9.1"
```

Comment:
Does this need to be here if it exists in the internals package-json?

Reply:
I'm removing all the EBNF parsing code from this PR for now, but you're right this `ebnf` package should only need to be installed in the `internals-js` package.